packages/quantic/force-app/main/default/lwc/quanticResultsPerPage/e2e/quanticResultsPerPage.e2e.ts

```diff
@@ -0,0 +1,97 @@
+import {testSearch, testInsight} from './fixture';
+import {
+  useCaseEnum,
+  useCaseTestCases,
+} from '../../../../../../playwright/utils/useCase';
+
+const fixtures = {
+  search: testSearch,
+  insight: testInsight,
+};
+
+const defaultChoices = [10, 25, 50, 100];
+const defaultInitialChoice = 10;
+
+useCaseTestCases.forEach((useCase) => {
+  let test = fixtures[useCase.value];
+
+  test.describe(`quantic results per page ${useCase.label}`, () => {
+    test.describe('with default options', () => {
+      test('should load as expected', async ({resultsPerPage}) => {
+        await test.expect(resultsPerPage.component).toBeVisible();
+        test
+          .expect(await resultsPerPage.allResultsPerPageOptionsButtons.count())
+          .toBe(defaultChoices.length);
+        test
+          .expect(resultsPerPage.selectedResultsPerPageOption)
+          .toHaveText(defaultInitialChoice.toString());
+      });
```

Comment:
This looks like a unit that has already been tested in the unit tests file, I would remove it.

Reply:
Yea I know it's there "twice", I thought it still made sense to make sure it loaded correctly, that way it tests LWS/Locker, etc.

If a more complex test were to fail, it would be harder to know what the issue was, but if we simply have a "does the component load" test, I think it's easier to know that something went really wrong :) 

We don't have to do that for all components, but I think it's useful to have both a UT and a e2e test if the component simply loads.